packages/neon-core/src/sc/ContractParam.ts

```diff
@@ -16,22 +16,44 @@ export enum ContractParamType {
   Signature = 0x17, // TODO: Implement support
 
   Array = 0x20,
-  Map = 0x22, // TODO: Implement support
+  Map = 0x22,
 
   InteropInterface = 0x30, // TODO: Implement support
 
   Void = 0xff,
 }
 
+export type ContractParamMap = {
+  key: ContractParam;
+  value: ContractParam;
+}[];
+
+export type ContractParamMapJson = {
+  key: ContractParamJson;
+  value: ContractParamJson;
+}[];
+
+export type ContractParamsMapLike = {
+  key: ContractParamLike;
+  value: ContractParamLike;
+}[];
+
 export interface ContractParamJson {
   type: string;
-  value?: string | boolean | number | ContractParamJson[] | null;
+  value?:
+    | string
+    | boolean
+    | number
+    | ContractParamJson[]
+    | ContractParamMapJson

```

Comment:
I believe this should be `ContractParamMapJson[]` (or `ContractParamMapEntryJson[]` if renamed)